src/libraries/System.Private.Xml/src/System/Xml/Dom/DocumentXPathNavigator.cs

```diff
@@ -1282,19 +1289,22 @@ public override bool CheckValidity(XmlSchemaSet schemas, ValidationEventHandler
                 throw new InvalidOperationException(SR.XmlDocument_NoSchemaInfo);
             }
 
+            // TODO: should we guard against null in above condition instead?
```

Comment:
```suggestion
            // TODO-NULLABLE: should we guard against null in above condition instead?
```

please change all `TODO` into `TODO-NULLABLE`

Reply:
Done.